### ISSUE TITLE
Normalization step

### ISSUE THREAD

**Issue by aimirza** [xiaoli-dong/metaerg#15]
Does MetaErg perform any gene abundance normalization before mapping to higher functional Ontologies/pathways?  The library size of my metagenomic samples are highly variable and without normalizing there will likely be false positives when I perform statistical tests comparing functional pathways between groups.

**Comment by Finesim97** [comment]
No, if you look for usages of the `sample2depth` and `mdepth_values` values in  the search and calculated values the depth values are just added together for each step.

**Comment by aimirza** [comment]
May you explain how the Abundance (vs count) is calculated for each protein-coding gene function profile?

**Comment by danielkim617** [comment]
I also wonder how it is

**Comment by aimirza** [comment]
> No, if you look for usages of the `sample2depth` and `mdepth_values` values in the search and calculated values the depth values are just added together for each step.

I couldnt understand the codes
